pkg/processor/k8sprocessor/pod_association.go

```diff
@@ -23,15 +23,21 @@ import (
 	"go.opentelemetry.io/collector/client"
 	conventions "go.opentelemetry.io/collector/model/semconv/v1.5.0"
 	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.uber.org/zap"
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/k8sprocessor/kube"
 )
 
 // extractPodIds extracts IP and pod UID from attributes or request context.
 // It returns a value pair containing configured label and IP Address and/or Pod UID.
 // If empty value in return it means that attributes does not contains configured label to match resources for Pod.
-func extractPodID(ctx context.Context, attrs pcommon.Map, associations []kube.Association) (podIdentifierKey string, podIdentifierValue kube.PodIdentifier) {
-	connectionIP := getConnectionIP(ctx)
+func extractPodID(
+	logger *zap.Logger,
+	ctx context.Context,
```

Comment:
nit: ctx as convention should be passes as first parameter

Reply:
I believe there is only rule according to ctx :sweat_smile:, but I don't have any document behind that :upside_down_face: